src/components/shared/Modal.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
I checked everywhere in your files and you aren't using this file, what's the purpose of it?

Reply:
Oooh I am sorry, 
it comes up after making rebase. 
I am gonna remove it.